xrpl/utils/ensure_classic_address.py

```diff
@@ -0,0 +1,35 @@
+"""Classic address helper utils."""
+
+from xrpl.constants import XRPLException
+from xrpl.core.addresscodec import is_valid_xaddress, xaddress_to_classic_address
+
+
+def ensure_classic_address(account: str) -> str:
```

Comment:
If this is being added as a helper method, it should really be in the address codec

Reply:
It exists in https://github.com/XRPLF/xrpl.js/blob/develop/packages/xrpl/src/sugar/utils.ts for xrpl.js. 

Question:
Is the address codec code the same between xrpl-py and xrpl.js? Are there people working on both?

Reply:
We can move it for xrpl-py - I think that's a better place for it to live. All the other code that involves classic vs x-addresses is there.

Reply:
The two libraries don't need to be _exactly_ the same.